programs/fileio.c

```diff
@@ -2147,6 +2244,82 @@ FIO_fwriteSparseEnd(const FIO_prefs_t* const prefs, FILE* file, unsigned storedS
     }   }
 }
 
+static void FIO_writePoolSetDstFile(write_pool_ctx_t *ctx, FILE* dstFile) {
+    assert(ctx!=NULL);
+    // We can change the dst file only if we have finished writing
```

Comment:
nit: These need to be `/* */` comments. `//` comments are C99.

Reply:
Fixed